docs/platform/howto/use-aws-privatelinks.rst

```diff
@@ -0,0 +1,205 @@
+     an allowed role can connect from any VPC.
+
+   -  Using the Aiven CLI, run the following command including your AWS
+      account ID, the access scope, and the name of your Aiven service:
+
+      ::
+
+         $ avn service privatelink aws create --principal arn:aws:iam::$AWS_account_ID:$access_scope $Aiven_service_name
+
+      For example:
+
+      ::
+
+         $ avn service privatelink aws create --principal arn:aws:iam::012345678901:user/mwf my-kafka
+
+   -  Using the Aiven web console:
+
+      #. Log in to the Aiven web console and select the service that you
+         want to use.
+
+      #. Select the **Network** tab and click **Create Privatelink** .
+
+      #. | Enter the Amazon Resource Names (ARN) for the principals that
+           you want to use, then click **Create** .
+
+   | This creates an AWS network load balancer dedicated to your Aiven
+     service and attaches it to an AWS VPC endpoint service that you can
+     later use to connect to your account's VPC endpoint.
+   | The PrivateLink resource stays in the initial ``creating`` state
+     for up to a few minutes while the load balancer is being launched.
+     After the load balancer and VPC endpoint service have been created,
+     the state changes to ``active`` and the ``aws_service_id`` and
+     ``aws_service_name`` values are set.
+
+#. In the AWS CLI, run the following command to create a VPC endpoint:
+
+   ::
+
+      $ aws ec2 --region eu-west-1 create-vpc-endpoint --vpc-endpoint-type Interface --vpc-id $your_vpc_id --subnet-ids $space_separated_list_of_subnet_ids --security-group-ids $security_group_ids --service-name com.amazonaws.vpce.eu-west-1.vpce-svc-0b16e88f3b706aaf1
+
+   | 
+   | Replace the ``--service-name`` value with the value shown next to
+     **Network** > **AWS service name** in the Aiven web console or by
+     running the following command in the Aiven CLI:
+
+   ::
+
+      $ avn service privatelink aws get aws_service_name
+
+   | 
+   | Note that for fault tolerance, you should specify a subnet ID for
+     each availability zone in the region. The security groups determine
+     the instances that are allowed to connect to the endpoint network
+     interfaces created by AWS into the specified sub-nets.
+   | Alternatively, you can create the VPC endpoint in the AWS web
+     console under **VPC** > **Endpoints** > **Create endpoint** . See
+     the `AWS
+     documentation <https://docs.aws.amazon.com/vpc/latest/userguide/vpce-interface.html#create-interface-endpoint>`__
+     for details.
+   | **Note:** For Aiven for Apache Kafka® services, the security group
+     for the VPC endpoint must allow ingress in the port range
+     ``10000-31000`` to accommodate the pool of Kafka broker ports used
+     in our PrivateLink implementation.
+   | It takes a while before the endpoint is ready to use as AWS
+     provisions network interfaces to each of the sub-nets and connects
```

Comment:
```suggestion
     provisions network interfaces to each of the subnets and connects
```

Reply:
Vale does not recognize 'subnets' as a valid word, that is why I use sub-nets on the first place. What  is the best solution reword as sub-nets or use literal markup ``subnets``